cg/cli/demultiplex/finish.py

```diff
@@ -79,21 +43,8 @@ def finish_flow_cell(
 def finish_all_cmd(context: CGConfig, dry_run: bool):
     """Command to finish up all demultiplexed flow cells."""
 
-    demux_post_processing_api_temp: DemuxPostProcessingAPI = DemuxPostProcessingAPI(config=context)
-    demux_post_processing_api_temp.set_dry_run(dry_run=dry_run)
-    is_error_raised: bool = demux_post_processing_api_temp.finish_all_flow_cells_temp()
+    demux_post_processing_api: DemuxPostProcessingAPI = DemuxPostProcessingAPI(config=context)
```

Comment:
This might be an instance where we can skip the type hint and named parameter for readability

Reply:
addressed in another pr